### PR TITLE
Updated hammer_cli_foreman_discovery-1.0.0

### PR DESCRIPTION
**Pull request by lzap**
* [x] Nightly
* [x] 1.15

Thanks

---

### CODE REVIEW

**Comment by domcleal** [comment]
[test] as Koji appears to be available now.

**Comment by dLobatog** [comment]
[test]

File: rubygem-hammer_cli_foreman_discovery/rubygem-hammer_cli_foreman_discovery.spec [hunk 1]
```diff
@@ -8,8 +8,8 @@
 
 Summary: Foreman discovery commands for Hammer CLI
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.0.3
-Release: 2%{?dist}
+Version: 1.0.0
+Release: 0%{?dist}
```
Comment (dLobatog) [inline rubygem-hammer_cli_foreman_discovery/rubygem-hammer_cli_foreman_discovery.spec]: This value is normally 1

Comment (ehelms) [inline rubygem-hammer_cli_foreman_discovery/rubygem-hammer_cli_foreman_discovery.spec]: @lzap If you revisit this I can get this merged and built

Comment (lzap) [inline rubygem-hammer_cli_foreman_discovery/rubygem-hammer_cli_foreman_discovery.spec]: Sorry I am late on github, pushed.

**Pull request merged**